docs/maintain-guides-how-to-build-validator-community.md

```diff
@@ -0,0 +1,20 @@
+---
+id: maintain-guides-how-to-build-validator-community
+title: Build a Community for Your Validator
+sidebar_label: Build a Community for Your Validator
+description: Build a Community for Your Validator
+---
+
+Validators are welcome to promote your node in the channels listed below or other social media
+platforms to let more people know about what is about your validator, what have you done for the
+network, why nominators should support your work. There are multiple ways to help you to get
```

Comment:
```suggestion
network, and why nominators should support your work. There are multiple ways to help you to get
```
I'm not sure if you need both `what you have done for the network` and `validator stats`, are they the same thing?